boards/arm/efr32bg_sltb010a/efr32bg_sltb010a.dts

```diff
@@ -43,6 +43,31 @@
 	};
 };
 
+&pinctrl {
+	compatible = "silabs,gecko-pinctrl";
+        /* configuration for uart0 device, default state */
+        usart0_default: usart0_default {
+		group1 {
+			/* configure PE.11 as UART_RX and PE.10 as UART_TX */
+			psels = <GECKO_PSEL(UART_TX, GECKO_PORT_E, GECKO_PIN(10))>,
+				<GECKO_PSEL(UART_RX, GECKO_PORT_E, GECKO_PIN(11))>,
+				<GECKO_LOC(UART_LOC, GECKO_LOCATION(0))>,
+				<GECKO_SPEED(UART_SPEED, 115200)>;
```

Comment:
how is UART speed related to pinctrl? I think this should be a property of the `uart` device node itself.

Reply:
Agreed, removed